p2panda-rs/src/operation/operation_value.rs

```diff
@@ -9,6 +11,9 @@ pub enum OperationValue {
     /// Boolean value.
     Boolean(bool),
 
+    /// Bytes value.
+    Bytes(ByteBuf),
```

Comment:
Yeh, this would work! I had it as a `Vec<u8>` before (for the reason you mention) but I wanted bytes to be `Uint8Array` everywhere in `p2panda-js` which meant this value being a ByteBuf. But with that macro we can have both :+1: